### PR TITLE
changes to incorporate Fquins suggestion on solar constant factor

### PR DESCRIPTION
**Pull request by passangd**
Changes to include Fuqin's suggestion on solar constant (mainly "SOLCON" and "LSUNFL" key in modtran_profile_json.py file and test datasets (json filess)  in env tests module.

### CODE REVIEW

**Review by uchchwhash — APPROVED** [review]
I have absolutely no idea what this is!

Approved! :smile:  :+1:

File: wagl/acquisition/landsat.py [hunk 1]
```diff
@@ -170,6 +172,42 @@ def __init__(self, pathname, uri, acquisition_datetime, band_name='BAND 1',
...
+        if not esun:
+            return super(Landsat8Acquisition, self).radiance_data(window, out_no_data, esun)
+
+        data = self.data(window=window)
+
+        # check for no data
+        no_data = self.no_data if self.no_data is not None else 0
+        nulls = data == no_data
+
+        print(data)
```
Comment (sixy6e) [inline wagl/acquisition/landsat.py]: I think we can remove the print statements

File: wagl/acquisition/landsat.py [hunk 2]
```diff
@@ -170,6 +172,42 @@ def __init__(self, pathname, uri, acquisition_datetime, band_name='BAND 1',
...
+
+        gain = self.reflectance_mult
+        bias = self.reflectance_add
+
+        print(gain, bias, esun)
```
Comment (sixy6e) [inline wagl/acquisition/landsat.py]: and again

File: wagl/standardise.py [hunk 3]
```diff
@@ -388,14 +398,16 @@ def card4l(level1, granule, workflow, vertices, method, pixel_quality, landsea,
...
+                                          filter_opts, normalized_solar_zenith,
+                                          esun_values[acq.band_name])
 
             # pixel quality
             sbt_only = workflow == Workflow.SBT
             if pixel_quality and can_pq(level1, acq_parser_hint) and not sbt_only:
                 run_pq(level1, res_group, landsea, res_group, compression, filter_opts, AP.NBAR, acq_parser_hint)
                 run_pq(level1, res_group, landsea, res_group, compression, filter_opts, AP.NBART, acq_parser_hint)
 
+        print(esun_values)
```
Comment (sixy6e) [inline wagl/standardise.py]: remove the print statement

**Review by sixy6e — APPROVED** [review]
Everything looks good.

Just remove the print statements.

**Comment by sixy6e** [comment]
Awesome!

Merge away

**Pull request merged**